File: VRSampleApp/app/src/main/java/com/ooyala/sample/screen/IMAVideoFragment.java
package com.ooyala.sample.screen;

import com.ooyala.android.imasdk.OoyalaIMAManager;
import com.ooyala.android.skin.OoyalaSkinLayout;

public class IMAVideoFragment extends VideoFragment {

  @Override
  public void applyADSManager(OoyalaSkinLayout skinLayout) {
    final OoyalaIMAManager imaManager = new OoyalaIMAManager(player, skinLayout);
  }
}

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/IMAVideoFragment.kt
package com.ooyala.sample.screen

import com.ooyala.android.imasdk.OoyalaIMAManager
import com.ooyala.sample.VideoFragment
import com.ooyala.sample.utils.VideoData
import kotlinx.android.synthetic.main.video_fragment.*

class IMAVideoFragment : VideoFragment() {


  override fun initAdManager() {
    val imaManager = OoyalaIMAManager(player, playerSkinLayout)
  }
}